tests/josegonzalez/Dotenv/LoaderTest.php

```diff
@@ -141,6 +141,7 @@ public function testParse()
             'STRING_FALSE' => 'false',
             'STRING_EMPTY' => '',
             'STRING_EMPTY_2' => '',
+            '01SKIPPED'=>'skipped',
```

Comment:
Will that be for 2.x? If so, I would rather wait if possible than merge this as is as both are bc breaks. *shrug* up to you though :)